components/Balance/Balance.tsx

```diff
@@ -0,0 +1,18 @@
+import styles from "../Balance/Balance.module.css";
+
+export type BalanceProps = {
+  totalValue: string;
+  returnValue: string;
```

Comment:
If these are numbers, you could/should use `number` type.

The suffix `Value` might be redundant.